simplepath/utils.py

```diff
@@ -50,3 +51,11 @@ def deepvars(data_object):
         return object_dict
     else:
         return data_object
+
+
+def divide(a, b):
```

Comment:
we can mention that in the lookup doc that it works like in Python3. Plus in Python2 people can do:

```python
from __future__ import division
```